feat: nodemail e fix: pagina de contato

EduLeal opened the pull request.

EduLeal committed:
feat: nodemail

luizbrito6 commented on pages/contato/index.js:
```diff
@@ -0,0 +1,21 @@
+})
+
+async function teste(data) {
+  const response = await fetch("http://localhost:4652/send/text", {
```
@PedroFnseca poderia passar a URL da API hospedada para o @EduLeal trocar o a url local?

luizbrito6 replied:
@EduLeal achei aqui

Só trocar a url local para essa que está hospedada (primeiro parâmetro do fetch)
https://web-production-e061.up.railway.app/send/text

EduLeal committed:
fix: pagina de contato

Atualização da página de contato no HTML e CSS, o CSS ficou um pouco grande mas a maior parte é dando o caminho das divs/tags

EduLeal committed:
fix: fetch

EduLeal committed:
fix: url

EduLeal committed:
fix: icon nas paginas contato e main
